### DISCUSSION TITLE
How to display device toolbar

### DISCUSSION THREAD

**Discussion by maxigit** [qutebrowser/qutebrowser#5677] [Q&A] [answered]
When I launch the devtool I can't see the device toolbar that you'll find on Chrome for example. This bar allows to render  a page for different device size, usefull to test responsive devices. Once the devtool window is open I can set "hide media queries" or "hide ruler" (using the run command menu) which are displayed in the device tool bar but there is no command to toggle the media toolbar. Is there way to toggle it, or is just not implemented yet.

**Comment by The-Compiler** [comment, marked as answer]
Not implemented upstream in Qt: [[QTBUG-85171] Missing features in inspector/devtools - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-85171)

Might start working in Qt 5.15.1 according to the comments, but I haven't tested yet.

**Reply by khaalidrafi** [reply]
> Not implemented upstream in Qt: [[QTBUG-85171] Missing features in inspector/devtools - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-85171)
> 
> Might start working in Qt 5.15.1 according to the comments, but II haven't tested yet.

Is it still unavailable?

**Reply by The-Compiler** [reply]
Try it and see? Doesn't seem to be there.

**Comment by maxigit** [comment]
Thanks for the quick answer.
Is there a way I can test it  (by upgrading to Qt 5.15.1) or shouldn I wait for a new release of Qutebrowser ?

**Reply by The-Compiler** [reply]
I tried with Qt 5.15.1 (using the Qt binary installer, `scripts/dev/build_pyqt_wheel.py` and `scripts/mkvenv.py`) and I still don't see it. However, this is only a preview of Qt 5.15.1, so maybe things will still change.

**Comment by maxigit** [comment]
I tried with WebKit but it doesn't have this feature either. I just have to
wait then ;-).
Thanks

**Comment by khaalidrafi** [comment]
> Try it and see? Doesn't seem to be there.

I tried it already. There is an option called screencast. But, I'm
unsure how to use that.

-- 
Khalid Rafi
Sent with Emacs
